src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -449,7 +460,7 @@ protected int doWork() {
             if (SequenceUtil.isNoCall(base)) continue;
 
             // add to the collector
-            collector.addInfo(info, ref);
+            collector.addInfo(info);
```

Comment:
Don't need to change it. But `ref` argument wasn't used in `addInfo`. IntelliJ said we don't call this method anywhere else in the project, so I figured it's cleaner to remove it. If you'd rather have it back, I will revert the change.